test/Integration/IntegrationTestBase.cs

```diff
@@ -57,7 +59,7 @@ public class IntegrationTestBase : IDisposable
         /// </summary>
         protected int Port { get; private set; } = 7071;
 
-        public IntegrationTestBase(ITestOutputHelper output)
+        public IntegrationTestBase(ITestOutputHelper output = null)
```

Comment:
Make this and TestOutput nullable

Reply:
I think ITestOutputHelper is already nullable. When I add Nullable<ITestOutputHelper> I get the following error:
`The type 'ITestOutputHelper' must be a non-nullable value type in order to use it as parameter 'T' in the generic type or method 'Nullable<T>'`

Reply:
What I meant is `ITestOutputHelper? output` - but I forgot that since this is targeting netstandard2.0 we can't enable that. So nevermind!